spec/controllers/address_controller_spec.rb

```diff
@@ -2,28 +2,23 @@
 
 require "rails_helper"
 
-RSpec.describe AddressController, :member, type: :controller do
-  let!(:current_app) do
-    SnapApplication.create!(
-      user: member,
-      street_address: "123 Fake St",
-      city: "Springfield",
-      county: "Genesee",
-      zip: "12345",
-      state: "MI",
-    )
-  end
-
+RSpec.describe AddressController, type: :controller do
```

Comment:
Minor: I believe we don't need to specify the spec type (e.g. `type: :controller`) because we've set `config.infer_spec_type_from_file_location` [here][1].

[1]: https://github.com/codeforamerica/michigan-benefits/blob/6fc3d9f77d619a12c19b7c7db4dbf01cbe939a93/spec/rails_helper.rb#L65